Code/Mantid/Framework/Algorithms/test/ConvertToMatrixWorkspaceTest.h

```diff
@@ -83,8 +89,8 @@ class ConvertToMatrixWorkspaceTest : public CxxTest::TestSuite
     TS_ASSERT_EQUALS( in->getInstrument()->isParametrized(), out->getInstrument()->isParametrized());
     for (size_t i=0; i < out->getNumberHistograms(); i++)
```

Comment:
Not your code exactly, but something to think about. You are exercising your test asserts inside a loop. If one of these tests fail. How do you know which loop iteration is responsible? You'd have to debug. My preference would be to use the TSM_ASSERT instead of TS_ASSERT, and use the message to format a string which contains the loop index. That way you'll see which iteration is responsible for the failure.